packages/validator-query/index.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const loadRules = require('./lib/rules');
+const AmpValidatorQuery = require('./lib/AmpValidatorQuery');
+
+let cached = null;
+
+async function load(opt) {
+  opt = opt || {};
+  if (!opt.noCache && cached) {
+    return cached;
+  }
+
+  let rules = opt.rules;
+  delete opt.rules;
+
+  if (!rules) {
+    rules = await loadRules(opt);
+  }
+
+  cached = new AmpValidatorQuery(rules);
+  return cached;
+}
+
+module.exports = { load };
```

Comment:
Seems to be some lint errors on this line:

```sh
❯ npm run lint

> root@ lint /Users/stillers/workspace/amp-toolbox
> eslint "**/*.js" --ignore-path .gitignore


/Users/stillers/workspace/amp-toolbox/packages/validator-query/index.js
  41:18  error  There should be no space after '{'   object-curly-spacing
  41:25  error  There should be no space before '}'  object-curly-spacing

✖ 2 problems (2 errors, 0 warnings)
  2 errors and 0 warnings potentially fixable with the `--fix` option.
```

(Not sure why Travis didn't run on the PR.)

Reply:
Argh, missed that one. Fixed.